Packs/Armis/Integrations/Armis/Armis.yml

```diff
@@ -116,6 +90,7 @@ script:
       required: false
       secret: false
     - default: false
+      defaultValue: 3 days
```

Comment:
Isn't this value 30 days in the code? 
![image](https://user-images.githubusercontent.com/50294648/110235065-bc5b6580-7f36-11eb-8391-298d866d4928.png)